src/geo/layer_definition.js

```diff
@@ -2,21 +2,10 @@
 function MapBase(options) {
```

Comment:
This file should not be used anymore, so we should be able to delete it pretty soon.